fido/src/main/java/com/yubico/yubikit/fido/ctap/Ctap2Session.java

```diff
@@ -644,16 +636,32 @@ private static InfoData fromData(Map<Integer, ?> data) {
                     data.containsKey(RESULT_TRANSPORTS)
                             ? (List<String>) data.get(RESULT_TRANSPORTS)
                             : Collections.emptyList(),
-                    (List<PublicKeyCredentialParameters>) data.get(RESULT_ALGORITHMS),
-                    (Integer) data.get(RESULT_MAX_SERIALIZED_LARGE_BLOB_ARRAY),
-                    (Boolean) data.get(RESULT_FORCE_PIN_CHANGE),
-                    (Integer) data.get(RESULT_MIN_PIN_LENGTH),
+                    data.containsKey(RESULT_ALGORITHMS)
+                            ? (List<PublicKeyCredentialParameters>) data.get(RESULT_ALGORITHMS)
+                            : Collections.emptyList(),
+                    data.containsKey(RESULT_MAX_SERIALIZED_LARGE_BLOB_ARRAY)
```

Comment:
For maxSerializedLargeBlobArray and maxCredBlobLength the spec states that if the key is not present the feature is not supported. 1024 and 32 are minimum values IF the feature is supported. Because of this I think it would be more appropriate to use 0 for both default values.

Reply:
fixed